src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/convolution_backprop_data.hpp

```diff
@@ -74,24 +74,24 @@ const std::vector<size_t> numOutChannels_Planar = {6};
 const std::vector<size_t> numOutChannels_Blocked = {64};
 
 /* ============= Deconvolution params (2D) ============= */
-const std::vector<std::vector<size_t>> kernels2d = {{3, 3}, {1, 1}};
+const std::vector<std::vector<size_t>> kernels2d = {{3, 3}, {2, 2}};
```

Comment:
Could you please explain the reason behind removing {1,1} kernels from the test?
Why should we prefer {2,2} kernels over {1,1} in the SL tests?

Reply:
Will try adding 1x1 case into and see.